pkg/envoy/server_test.go

```diff
@@ -641,8 +611,8 @@ var ExpectedPerPortPoliciesKafka = []*cilium.PortNetworkPolicy{
 		Port:     9092,
 		Protocol: envoy_config_core.SocketAddress_TCP,
 		Rules: []*cilium.PortNetworkPolicyRule{{
-			RemotePolicies: []uint64{1001, 1002},
-			L7Proto:        "kafka",
+			// RemotePolicies: []uint64{1001, 1002}, // Effective wildcard due to only one selector in the policy
```

Comment:
Is this code still meant to be commented out?

Reply:
Yes. 1001 and 1002 are the only allowed remotes, but we (now) send it to Envoy as an L3 wildcard as the enforcement is already done in bpf and there are no L3-dependent L7 rules. The comment could help someone trying to work with this test case in future.